web/js/jquery.jmri.js

```diff
@@ -955,7 +955,7 @@
                                 if (h) {
                                     h.call(this, o);
                                 } else if (!o.type) {
-                                    jmri.log("ERROR: missing type property in " + o);
+                                    console.error("ERROR: missing type property in " + o);
```

Comment:
Note: You *cannot* rely on the console existing--there is no reason not to expect `console.error` to crash out the JavaScript completely on user's devices, so please revert this, or come up with a reliable alternative.

There is *zero* benefit to writing code to making unit testing easy if if crashes out entirely for users.